CONTRIBUTING.md

```diff
@@ -1,7 +1,28 @@
-# Contributing
+# Contributing to panther-analysis
 
-Please follow the [Code of Conduct](https://github.com/panther-labs/panther-analysis/blob/main/CODE_OF_CONDUCT.md)
-in all of your interactions with the project.
+Thank you for your interest in contributing to Panther's open-source ruleset!  We value and appreciate all contributions, whether they are bug reports, feature requests, or detection rule contributions.
+
+## What Makes a Good Detection?
+
+Please familiarize yourself with these helpful resources on writing high-quality Panther rules.  Especially excellent contributions will be considered for seasonal prizes!
+
+- Jack's blog on [The Anatomy of a High Quality SIEM Rule](https://jacknaglieri.substack.com/p/hq-siem-rules)
```

Comment:
"Jack" is a little too familiar. sugg: The blog post Panther's founder, Jack Naglieri, wrote on [The ...